package.json

```diff
@@ -2,12 +2,13 @@
   "name": "control-panel",
   "version": "1.3.4",
   "description": "embeddable panel of inputs for parameter setting",
-  "main": "index.js",
+  "main": "build/control-panel.js",
   "scripts": {
     "start": "watchify-server example.js | garnish",
     "test": "standard && browserify test.js | smokestack | tap-spec",
-    "build": "browserify demo/index.js -o demo/bundle.js",
-    "deploy": "npm run build && surge demo control-panel.surge.sh"
+    "build": "browserify -s control index.js -o build/control-panel.js",
```

Comment:
Would it be appropriate to add the `es2040` browserify transform to this, i.e.:

```javascript
browserify -s control index.js -o build/control-panel.js -t es2040
```

This repo uses template strings which at this point are mostly supported, but my iPad, for one, fails with const and template strings so that it's probably best to just transpile to ES5 while we're at it.

Reply:
Sure thing, makes sense